src/ArmTemplates/Common/Templates/ApiOperations/ApiOperationRepresentation.cs

```diff
@@ -14,5 +15,7 @@ public class ApiOperationRepresentation
         public string TypeName { get; set; }
 
         public string Sample { get; set; }
+
+        public IDictionary<string, ParameterExampleContract> Examples { get; set; }
```

Comment:
In [documentation](https://github.com/Azure/azure-resource-manager-schemas/blob/3cc70eddf0341c5336f5ffd38c44c14deb9ed4e4/schemas/2021-08-01/Microsoft.ApiManagement.json#L5484-L5486) we have it possible also as a `string`. Please double-check it with somebody else (that docs are not lying) :)

In other things review LGTM!

Reply:
Thank you for mentioning that! Description of the PR is updated